torch_geometric/nn/models/mlp.py

```diff
@@ -76,18 +79,25 @@ def __init__(
         num_layers: Optional[int] = None,
         dropout: float = 0.,
         act: str = "relu",
-        batch_norm: bool = True,
         act_first: bool = False,
         act_kwargs: Optional[Dict[str, Any]] = None,
-        batch_norm_kwargs: Optional[Dict[str, Any]] = None,
+        norm: Optional[str] = 'batch_norm',
+        norm_kwargs: Optional[Dict[str, Any]] = None,
         plain_last: bool = True,
         bias: bool = True,
-        relu_first: bool = False,
+        **kwargs,
     ):
         super().__init__()
 
-        act_first = act_first or relu_first  # Backward compatibility.
-        batch_norm_kwargs = batch_norm_kwargs or {}
+        # Backward compatibility:
+        act_first = act_first or kwargs.get("relu_first", False)
+        batch_norm = kwargs.get("batch_norm", None)
+        if batch_norm is not None and isinstance(batch_norm, bool):
+            warnings.warn("Argument `batch_norm` will be deprecated, "
```

Comment:
I'm wondering if we should decide on a version in which it will be depreciated and note that here? WDYT?

Reply:
I am not sure when we want to fully deprecate it though. @rusty1s any idea on this?